src/widget.js

```diff
@@ -1,33 +1,30 @@
-const getFunctionTypeParamsString = params => {
-  if (!params) {
-    return "";
-  }
+const getFunctionTypeParamsString = (params = []) => {
   return params
     .reduce((acc, param) => `${acc}${param.name}: any, `, "")
     .slice(0, -2);
 };
 
+const getEventHandlersTypes = eventHandlers => {
+  return Object.keys(eventHandlers).reduce((acc, value) => {
+    return `${acc} ${value}(eventHandler: (event: any) => void );\n`;
+  }, "");
+};
+
 const getMembersTypes = members => {
   return Object.keys(members).reduce((acc, value) => {
```

Comment:
i think it would be clearer if instead of reduce, you would map and at the end join with "\n"